dojo/tools/anchore_grype/parser.py

```diff
@@ -70,3 +166,21 @@ def _convert_severity(self, val):
             return "Info"
         else:
             return val.title()
+
+    def get_cve(self, vuln_id, rel_id):
+        if vuln_id and vuln_id.startswith('CVE'):
+            return vuln_id
+        elif rel_id and rel_id.startswith('CVE'):
+            return rel_id
+        else:
+            return vuln_id
+
+    def get_cvss(self, cvss):
+        if cvss:
+            for cvss_item in cvss:
+                vector = cvss_item['vector']
+                cvss_objects = cvss_parser.parse_cvss_from_text(vector)
+                if len(cvss_objects) > 0 and type(cvss_objects[0]) == CVSS3:
+                    vector = cvss_item['vector']
```

Comment:
Line 181 and 184 are the same.

Reply:
Well spotted, I have removed the second line